src/auth/auth0ProviderWithHistory.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Auth0Provider } from '@auth0/auth0-react';
+
+const Auth0ProviderWithHistory = ({ children }) => {
+  const domain = process.env.REACT_APP_AUTH0_DOMAIN;
+  const clientId = process.env.REACT_APP_AUTH0_CLIENT_ID;
+
+  const history = useHistory();
+
+  const onRedirectCallback = appState => {
+    console.log(appState);
```

Comment:
let's remove this console.log(). I know we're still in dev mode here @paulstgermain might have wrote this and intended to remove it prior to production.